src/Components/UpcomingRow/UpcomingRow.tsx

```diff
@@ -98,8 +98,13 @@ export const TableRow: React.FunctionComponent<TableRowProps> = ({
               onToggle,
               expandId: 'composable-expandable-example',
             }}
+            className="drf-lifecycle__expand-column"
```

Comment:
I couldn't find any style with this name, haven't you forgotten to add `scss` file to the commit/is the className needed at all?

Reply:
Was going to use it but we can remove it for now